extras/zot-cache/values.yaml

```diff
@@ -34,10 +34,10 @@ giantswarm:
 resources:
   requests:
     cpu: 300m
-    memory: 1024Mi
+    memory: 1536Mi
   limits:
-    cpu: 500m
-    memory: 1024Mi
+    cpu: 800m
+    memory: 1536Mi
```

Comment:
I would recommend increasing the `GOMEMLIMIT` too - just below - by the same proportion, e.g. to `1200MiB`.